pkg/rebuild/rebuild/git.go

```diff
@@ -92,27 +91,19 @@ func FindTagMatch(pkg, version string, repo *git.Repository) (commit string, err
 func allTags(repo *git.Repository) (tags []string, err error) {
 	ri, err := repo.Tags()
 	if err != nil {
-		return
+		return nil, err
 	}
 	var ref *plumbing.Reference
 	for {
 		ref, err = ri.Next()
 		if err == io.EOF {
 			break
 		} else if err != nil {
-			return
+			return nil, err
 		}
 		tags = append(tags, ref.Name().Short())
 	}
-	ti, err := repo.TagObjects()
-	if err != nil {
-		return
-	}
-	err = ti.ForEach(func(t *object.Tag) error {
-		tags = append(tags, t.Name)
-		return nil
-	})
-	return
```

Comment:
Was the previous version just including all tags twice?

Reply:
it looks like it, yes. it wouldn't have resulted in any changes in inference but it may have increased runtime by a tiny amount.